src/pages/SubmissionDetails.js

```diff
@@ -81,7 +81,7 @@ export const SubmissionDetails = ({ submissionId }) => {
       case 'CANCELING':
         return statusType.canceling
       default:
-        return statusType.unknown
+        return (differenceFromNowInSeconds(submissionDate) < 10) ? statusType.initializing : statusType.unknown
```

Comment:
TOL: Is 10 seconds enough? If yes, would it be worth adding a comment why 10 seconds is enough?